packages/protobuf/src/private/names.ts

```diff
@@ -89,7 +80,7 @@ export function localFieldName(protoName: string, inOneof: boolean) {
     // oneof member names are not properties, but values of the `case` property.
     return name;
   }
-  if (reservedObjectProperties[name] || reservedMessageProperties[name]) {
+  if (isReservedObjectProperty(name) || reservedMessageProperties[name]) {
     name = name + "$";
```

Comment:
I didn't update `reservedMessageProperties` like the other two (`reservedObjectProperties` and `reservedIndent`) in the name of keeping scope down, but if you'd like me to make them all look the same I'd be happy to.

Reply:
Let's make it consistent for sure.

I think this should probably boil down to `return safeMessageProprty(safeObjectProperty(name));` Please don't export safeMessageProperty().

Please check how that affects bundle size with `make bench`.

Reply:
```diff
  | code generator      | bundle size             | minified               | compressed         |
  |---------------------|------------------------:|-----------------------:|-------------------:|
+ | protobuf-es         | 86,823 b      | 36,972 b | 9,654 b |
- | protobuf-es         | 86,785 b      | 36,950 b | 9,642 b |
  | protobuf-javascript | 375,550 b  | 271,672 b | 43,769 b |
```

as of e4f43261f355c39a539528e0fa16409b80e8c044